scripts/in_container/run_migration_reference.py

```diff
@@ -184,9 +184,9 @@ def ensure_filenames_are_sorted(revisions):
 
 
 if __name__ == "__main__":
-    revisions = list(reversed(list(get_revisions())))
+    revisions = list(reversed(get_revisions()))
```

Comment:
```suggestion
    revisions = reversed(get_revisions())
```

Reply:
Do we actually have to re-create new `revisions` object three times here?

Reply:
I’m not following, this does not recommend you to do that.

Reply:
Correct. Will change that.